src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -124,11 +124,11 @@ object SbtGit {
     scmInfo := {
       val user = """(?:[^@\/]+@)?"""
       val domain = """([^\/]+)"""
-      val gitPath = """(.*)\.git\/?"""
+      val gitPath = """(.*)(\.git)?\/?"""
```

Comment:
You can use ?: for a non-binding match.

```suggestion
      val gitPath = """(.*)(?:\.git)?\/?"""
```

Then you don't have to care about the match below.

Reply:
Neat, thx!